src-db/database/sourcedata/AD_MESSAGE.xml

```diff
@@ -108,6 +108,18 @@
 <!--1E603ECA43C1467DB1A2E9EDD6F3438F-->  <ISINCLUDEINI18N><![CDATA[N]]></ISINCLUDEINI18N>
 <!--1E603ECA43C1467DB1A2E9EDD6F3438F--></AD_MESSAGE>
 
+<!--26A9FCB5546E46BFAF6F0A9F36F80635--><AD_MESSAGE>
+<!--26A9FCB5546E46BFAF6F0A9F36F80635-->  <AD_MESSAGE_ID><![CDATA[26A9FCB5546E46BFAF6F0A9F36F80635]]></AD_MESSAGE_ID>
+<!--26A9FCB5546E46BFAF6F0A9F36F80635-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--26A9FCB5546E46BFAF6F0A9F36F80635-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--26A9FCB5546E46BFAF6F0A9F36F80635-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--26A9FCB5546E46BFAF6F0A9F36F80635-->  <VALUE><![CDATA[ETDEP_Utility-Class-Cannot-Instantiate]]></VALUE>
```

Comment:
Change the "-" in this message search key to "_" for it to follow the same naming conventions as the rest of messages